ipas_default.config

```diff
@@ -92,6 +92,14 @@ tests:
 skips:
   [ 'B101', 'B102', 'B103', 'B104', 'B105', 'B106', 'B107', 'B108', 'B110', 'B112', 'B201', 'B501', 'B502', 'B503', 'B504', 'B505', 'B506', 'B507', 'B601', 'B602', 'B603', 'B604', 'B605', 'B606', 'B607', 'B608', 'B609', 'B610', 'B611', 'B701', 'B702', 'B703']
 
+
+# Added to exclude some path which are not actual source code for this project
```

Comment:
isn't it better to use `targets = ['otx']` instead of using `exclude_dirs`?
I don't know well about this, so it could be wrong.

Reply:
@eunwoosh I don't know either but I think it's just "include" vs "exclude" not "good" or "better". If we use "target" attribute for configuration, we need to add "tests" path as well.